addons/binding/org.openhab.binding.nest.test/META-INF/MANIFEST.MF

```diff
@@ -1,26 +1,50 @@
 Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
 Bundle-ManifestVersion: 2
-Bundle-Name:  Nest Binding Tests
+Bundle-Name: Nest Binding Tests
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.openhab.binding.nest.test;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.3.0.qualifier
-Export-Package: org.openhab.binding.nest;uses:="org.eclipse.smarthome.te
- st"
+Export-Package: 
+ org.openhab.binding.nest;uses:="org.eclipse
+ .smarthome.test",

```

Comment:
Is this new, is this needed, why?

Reply:
Respect for phone reviews in gardens! :sunglasses::palm_tree:

I'll see if it can be removed. I must have added it in a desperate attempt to get OSGi tests running from Eclipse/Maven.

Reply:
Only one of those exports was necessary to get the OSGi tests working. It's also part the binding test skeleton [manifest](https://github.com/eclipse/smarthome/blob/82179a9a82c397409716d2bea1c607cea748f013/tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF#L8).